presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -133,6 +146,21 @@ public boolean isTerminated()
         return sourcePatterns.isEmpty();
     }
 
+    public static FunctionCall functionCall(String name, String... args)
```

Comment:
Per offline discussion, this is another one to hold pending the arrival of the new optimizer. The most obvious barrier to doing it now is that function calls aren't plan nodes, and neither are their parameters.